test/test_files/patcher_data/am2r/progressive_items.json

```diff
@@ -5000,7 +5000,7 @@
             ]
         },
         "rm_a3a05": {
-            "display_name": "Spazer Beam",
+            "display_name": "Spazer Beam Chamber",
```

Comment:
What exactly is this?

Reply:
used to tell the patcher on what string should be displayed with the "show room names" cosmetic option.